UnityProject/Assets/Scripts/ScriptableObjects/SingletonScriptableObject.cs

```diff
@@ -18,21 +18,21 @@ public static T Instance
 		{
 			get
 			{
-				if (!_instance && SOs.Instance != null)
+				if (_instance == null && (SOs.Instance == null) == false)
 				{
```

Comment:
!= Was fine here, I think. If anything the == false makes it harder to read than !=

Reply:
is unity okay with that?

Reply:
With != ? I don't see why not, it should be using the same equality comparison

Reply:
ok cool